regression/scripts/Vehicles/Concorde.py

```diff
@@ -254,7 +256,7 @@ def vehicle_setup():
     segment.root_chord_percent    = 14.5/14.5
     segment.dihedral_outboard     = 0.
     segment.sweeps.quarter_chord  = 63. * Units.deg
-    segment.thickness_to_chord    = 0.04
+    segment.thickness_to_chord    = 0.03
```

Comment:
Why were these thicknesses changed?

Reply:
Put back to normal, this accidentally got fed back in from an earlier version.